src/compiler/checker.ts

```diff
@@ -8824,7 +8824,8 @@ namespace ts {
             let targetStack: Type[];
             let maybeCount = 0;
             let depth = 0;
-            let expandingFlags = 0;
+            const enum ExpandingFlags { None = 0, Source = 1, Target = 1 << 1, Both = Source | Target }

```

Comment:
can you format this with the normal amount of newlines? the current position is still probably ok.